flink-runtime/src/main/java/org/apache/flink/runtime/state/ttl/TtlMapState.java

```diff
@@ -54,7 +54,13 @@
 
 	@Override
 	public UV get(UK key) throws Exception {
-		return getWithTtlCheckAndUpdate(() -> original.get(key), v -> original.put(key, v), () -> original.remove(key));
+		TtlValue<UV> ttlValue = getWrapped(key);
+		return ttlValue == null ? null : ttlValue.getUserValue();
+	}
+
+	private TtlValue<UV> getWrapped(UK key) throws Exception {
+		return getWrappedWithTtlCheckAndUpdate(
+			() -> original.get(key), v -> original.put(key, v), () -> original.remove(key));
```

Comment:
Is it necessary to add a private method here? 
TtlValue<UV> ttlValue = getWrappedWithTtlCheckAndUpdate(
			() -> original.get(key), v -> original.put(key, v), () -> original.remove(key));
return ttlValue == null ? null : ttlValue.getUserValue();
looks better to me.

Reply:
I added it for code reuse because `get` and `contains` needed the same code.